cfpb/consumerfinance.gov
# P4C: Add copy button to save & finish section at bottom of grad path tool

Adds another copy button to save & finish section available on each grad path tool screen.


## Additions

- Another copy button


## Changes

- Move copy form html to its own template
- Update event handlers to work with multiple copy buttons


## How to test this PR

1. Check out branch and run gulp
2. Navigate to grad path tool and click `save & finish later` link at bottom of first screen. Verify that copy button appears and works as expected.


## Screenshots

<img width="760" alt="Screen Shot 2021-04-21 at 1 03 17 PM" src="https://user-images.githubusercontent.com/778171/115614026-fcd23100-a2a1-11eb-8b48-1e3e5b9e8d70.png">



## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
